backend/bitsurance/bitsurance.go

```diff
@@ -83,11 +83,11 @@ func BitsuranceGetId(account accounts.Interface) (string, error) {
 }
 
 // BitsuranceURL returns the url for the Bitsurance widget for a given locale.
-func BitsuranceURL(lang, bitsuranceId string) string {
-	return widgetURL + lang + "/?id=" + bitsuranceId
+func BitsuranceURL(lang string) string {
```

Comment:
We don't need the bitsuranceId when calling the iframe, as we decided to use it only for the onramp of new accounts